### PR TITLE
Np 46392 get single keyword

### PR DESCRIPTION
**Pull request by torarnet**

### CODE REVIEW

File: cristin-commons/src/main/java/no/unit/nva/model/IdentifierWithLabels.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import static no.unit.nva.cristin.model.JsonPropertyNames.CONTEXT;
+import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.model.JsonPropertyNames.IDENTIFIER;
+import static no.unit.nva.cristin.model.JsonPropertyNames.LABELS;
+import static no.unit.nva.cristin.model.JsonPropertyNames.TYPE;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.Map;
+
+public class IdentifierWithLabels {
```
Comment (torarnet) [inline cristin-commons/src/main/java/no/unit/nva/model/IdentifierWithLabels.java:12]: can be extended by other types than keyword as it has a lot of common fields

File: cristin-keyword/src/test/resources/nvaFetchKeywordResponse.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (torarnet) [inline cristin-keyword/src/test/resources/nvaFetchKeywordResponse.json:1]: Went for this model for keyword. I also have a task using this model for keyword query and possibly country list

File: template.yaml [hunk 3]
```diff
@@ -579,6 +579,19 @@ Resources:
             Path: /keyword/
             Method: post
 
+  NvaCristinListCountriesFunction:
```
Comment (torarnet) [inline template.yaml:582]: just moved this to the bottom. the real change is keyword up above

File: cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/KeywordType.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import java.net.URI;
+import java.util.Map;
+import no.unit.nva.commons.json.JsonSerializable;
+import no.unit.nva.cristin.keyword.KeywordConstants;
+import no.unit.nva.model.IdentifierWithLabels;
+import no.unit.nva.utils.UriUtils;
+
+public class KeywordType extends IdentifierWithLabels implements JsonSerializable {
+
+    public static final String context = "https://bibsysdev.github.io/src/keyword-context.json";
```
Comment (anetteOlli) [inline cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/KeywordType.java:12]: Denne url'en returnerer 404.

**Review by anetteOlli — APPROVED** [review]

File: cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/KeywordType.java (hunk 4 above)
Comment (torarnet) [reply cristin-keyword/src/main/java/no/unit/nva/cristin/keyword/model/nva/KeywordType.java:12]: Det er ikke implementert enda. Noen må lage context objektet og legge det på den url

**Review by brinxmat — APPROVED** [review]

**Pull request merged**